src/test/resources/graphics/scenery/tests/examples/volumes/AmanatidesJumps.comp

```diff
@@ -0,0 +1,2278 @@
+#version 450
```

Comment:
This is a crazy-long piece of shader code. Is really all of this necessary? Also the file name is totally unintuitive.